site/specs/webrtc.yml

```diff
@@ -2421,6 +2520,201 @@ components:
           example: participant1
         deviceApiVersion:
           $ref: '#/components/schemas/deviceApiVersionEnum'
+    createDialogRequest:
+      type: object
+      description: The important things to specify if for the success of the dialog
+      properties:
+        dialogType:
+          $ref: '#/components/schemas/dialogType'
+        dialogStateData:
+          oneOf:
+            - $ref: '#/components/schemas/sipDialogRequestStateData'
```

Comment:
I suggest renaming this to createSipDialogRequest. It's only used for that and that not clear when you encounter it elsewhere

Reply:
good idea.  Naming is important - thanks - done